bammetrics/src/main/scala/nl/lumc/sasc/biopet/pipelines/bammetrics/BammetricsReport.scala

```diff
@@ -175,8 +177,8 @@ object BammetricsReport extends ReportBuilder {
                            statsPaths: Map[String, List[String]],
                            xKey: String,
                            yKey: String,
-                           pipeline: Either[Int, String],
-                           module: Option[Either[Int, String]],
+                           pipeline: \/[Int, String],
+                           module: Option[\/[Int, String]],
```

Comment:
This function (`writePlotFromSummary`) needs more documentation. It's not immediately clear what the arguments are, especially since we have `Int \/ String` here. Does the `Int` there denote error code, for example?